src/components/ec-currency-input/ec-currency-input.spec.js

```diff
@@ -45,6 +51,47 @@ describe('EcCurrencyInput', () => {
     expect(wrapper.element).toMatchSnapshot();
   });
 
+  describe('when the bottom note is defined', () => {
+    it('should render properly', () => {
+      const wrapper = mountCurrencyInput({ bottomNote: 'Bottom note message' });
+      expect(wrapper.findByDataTest('ec-currency-input__bottom-note').element).toMatchSnapshot();
+    });
+
+    describe('and it is a warning note', () => {
+      it('should render properly', () => {
+        const wrapper = mountCurrencyInput({
+          bottomNote: 'Bottom note message',
+          isWarning: true,
+        });
+        expect(wrapper.findByDataTest('ec-currency-input__bottom-note').element).toMatchSnapshot();
+      });
+
+      describe('and there is a warning message defined', () => {
+        it('should render properly', () => {
+          const warningMessage = 'Warning message';
+          const wrapper = mountCurrencyInput({
+            bottomNote: 'Bottom note message',
+            isWarning: true,
+            warningMessage,
+          });
+          expect(wrapper.findByDataTest('ec-currency-input__bottom-note').element).toMatchSnapshot();
+          expect(wrapper.findByDataTest('ec-currency-input__warning-tooltip').attributes('mocked-tooltip-content')).toBe(warningMessage);
+        });
+      });
+    });
```

Comment:
maybe also cover the case when there are both, the error prevails?

Reply:
yes, the error always prevails over the bottom note and the warning icon with the tooltip. I added another test case to check this scenario also. Thanks.